presto-docs/src/main/sphinx/release/release-0.290.rst

```diff
@@ -0,0 +1,109 @@
+* GET `/v1/info/state` to return INACTIVE state until the resource group configuration manager is fully initialized :pr:`23585`.
+* Make IcebergDistributedSmokeTestBase abstract :pr:`23580`.
+* Upgraded avro to version 1.11.4 :pr:`23868`.
+* Upgraded commons-codec to version 1.17.0 :pr:`23868`.
+* Upgraded commons-compress to version 1.26.2 :pr:`23868`.
+* Upgraded commons-io to version 2.16.0 :pr:`23794`.
+* Upgraded commons-io to version 2.16.1 :pr:`23868`.
+* Upgraded commons-lang3 to version 3.14.0 :pr:`23868`.
+* Upgraded protobuf-java to version 3.25.5 :pr:`23797`.
+* Upgraded protobuf-java-util to version 3.25.5 :pr:`23797`.
+
+Security Changes
+________________
+* Upgrade Postgres JDBC Driver to 42.6.1 :pr:`23710`.
+
+Hive Connector Changes
+______________________
+* Fix interpretation of ambiguous timestamps inside array, map, or row types for tables using TEXTFILE format to interpret the timestamps as the earliest possible unixtime for consistency with the rest of Presto.
+* Fix timestamps inside array, map, or row types for tables using TEXTFILE format to respect the ``hive.time-zone property``.
+
+Iceberg Connector Changes
+_________________________
+* Fix time-type columns to return properly when ``iceberg.parquet-batch-read-optimization-enabled`` is set to ``TRUE``. :pr:`23542`.
+* Fix to reduce drop time for Iceberg tables with deleted metadata in S3 storage. :pr:`23510`.
+* Add Iceberg metadata table $ref :pr:`23503`.
+* Add configuration property ``iceberg.rest.auth.oauth2.scope`` for OAUTH2 authentication in Iceberg's REST catalog :pr:`23884`.
+* Add iceberg.rest.auth.oauth2.uri configurable property :pr:`23739`.
+* Add procedure `rollback_to_timestamp` to rollback an iceberg table to a given point in time. :pr:`23559`.
+* Add support of UUID-typed columns :pr:`23627`.
+* Add support to query Iceberg table by branch/tag name :pr:`23539`.
+* Add table property `metrics_max_inferred_column` to configure the max columns number for which metrics are collected, and support `metrics_max_inferred_column` for Iceberg table with `PARQUET` format :pr:`23468`.
+* Support procedure fast_forward for iceberg :pr:`23589`.
+* Support timestamp without timezone in time travel expressions :pr:`23714`.
+* Support using named arguments in procedure `register_table` and `unregister_table` :pr:`12345`.
+
+MongoDB Connector Changes
+_________________________
+* Support varbinary data type in MongoDB (pr:`23386`).
+
+Mongodb Connector Changes
+_________________________
+* Add support for MongoDB ``ALTER TABLE`` statement. :pr:`23266`.
+
+SPI Changes
+___________
+* Add ``Partitioning``, ``PartitioningScheme``, ``PartitioningHandle``, ``PlanFragmentId``, ``StageExecutionDescriptor`` and ``SimplePlanFragment`` to the SPI . :pr:`23601`.
+
+Elasticsearch Changes
+_____________________
+* Improve handling of exceptions for empty tables in Elasticsearch :pr:`23850`.
+
+Orc/dwrf Changes
```

Comment:
```suggestion
ORC/DWRF Changes
```
Capitalization for consistency with line 92. 

This heading does not appear in the [Order of sections](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-sections) in the Release Notes Guidelines. I don't think that automatically makes it wrong, but I ask that other reviewers consider if this entry more properly belongs in **General Changes** or maybe another section in the Release Notes.